comps/llms/text-generation/tgi/llm.py

```diff
@@ -62,7 +62,10 @@ async def llm_generate(input: Union[LLMParamsDoc, ChatCompletionRequest, Searche
                 logger.info(f"[ SearchedDoc ] retrieved docs: {input.retrieved_docs}")
                 for doc in input.retrieved_docs:
                     logger.info(f"[ SearchedDoc ] {doc}")
-            prompt = ChatTemplate.generate_rag_prompt(input.initial_query, input.retrieved_docs[0].text)
+            docs = [doc.text for doc in input.retrieved_docs]
+            if logflag:
+                logger.info(f"[ SearchedDoc ] docs: {docs}")
```

Comment:
Is this line duplicated with line 64?

Reply:
That's true. thanks, will delete line 64